src/Compilers/CSharp/Test/Semantic/Semantics/DelegateTypeTests.cs

```diff
@@ -13927,6 +13927,20 @@ public abstract class MulticastDelegate : Delegate { }
                 Diagnostic(ErrorCode.ERR_PredefinedTypeNotFound, "2m").WithArguments("System.Decimal").WithLocation(2, 24));
         }
 
+        [Fact]
+        public void ParamsArray_MissingParamArrayAttribute()
+        {
+            var source = """
+                var lam = (params int[] xs) => xs.Length;
+                """;
+            var comp = CreateCompilation(source);
+            comp.MakeTypeMissing(WellKnownType.System_ParamArrayAttribute);
+            comp.VerifyDiagnostics(
+                // (1,25): error CS0656: Missing compiler required member 'System.ParamArrayAttribute..ctor'
```

Comment:
>// (1,25): error CS0656: Missing compiler required member 'System.ParamArrayAttribute..ctor' [](http://example.com/codeflow?start=16&length=92)

Is the error goin to be reported even when predefined delegate type without `params` is used as the target? Something like ```Func<int[], int> lam = (params int[] xs) => xs.Length;```?